_docs/tasks/security/mutual-tls.md

```diff
@@ -24,6 +24,9 @@ This task assumes you have a Kubernetes cluster:
 Note to choose "enable Istio mutual TLS Authentication feature" at step 5 in
 "[Installation steps]({{home}}/docs/setup/kubernetes/quick-start.html#installation-steps)".
 
+
+Note: from 0.7, you can use [authentication policy]({{home}}/docs/concepts/security/authn-policy.html) to config mTLS for all/selected services in a namespace (repeated for all namespaces to get global setting). See [authentication policy task]({{home}}/docs/tasks/security/authn-policy.html)
```

Comment:
Remove "Note:" and instead start the line with > which will create a special indented block in the output.

from 0.7 -> Starting with Istio 0.7,

Reply:
Done